gtsfm/runner/run_vis_tool.py

```diff
@@ -0,0 +1,22 @@
+"""Runner for the visualization tool only (back-end results are not desired or required).
+React always runs the server on http://localhost:3000/ locally.
+
+Author: Hayk Stepanyan
+"""
+
+import os
+
+from pynpm import NPMPackage
+from pathlib import Path
+
+VIS_TOOL_PATH = Path(__file__).resolve().parent.parent.parent / "rtf_vis_tool"
+
+
+def run_vis_tool():
```

Comment:
could you move this to a util? we don't want the runner base class depending on another runner.